ale/base/base.py

```diff
@@ -342,13 +408,19 @@ def projection(self):
                 with tempfile.NamedTemporaryFile() as tmp:
                     tmp.write(pvl.dumps(self._file)) 
 
-                    geodata = gdal.Open(tempfile.name)
+                    try:
+                        geodata = gdal.Open(tempfile.name)
+                    except:
+                        geodata = None
             else: 
                 # should be a path
                 if not os.path.exists(self._file): 
                     self._projection = "" 
                 else: 
-                    geodata = gdal.Open(self._file)
+                    try:
+                        geodata = gdal.Open(self._file)
+                    except:
+                        geodata = None
```

Comment:
It may be worth reporting the exception to the user as a warning. When working with pds3 labels I was running into issues where the label was pointing to a data file causing the gdal.Open to fail but the label could be used as valid input for a driver.